docker-action.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Does this script have to be in the root of the repo. Keeping th root of the repo uncluttered is better